src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final int NumberOfBalls = 3;
```

Comment:
Create a constant means that you should make a variable static final on class level